resources/helm/v1.1/istio_cni/templates/daemonset.yaml

```diff
@@ -62,8 +62,8 @@ spec:
       containers:
         # This container installs the Istio CNI binaries
         # and CNI network config file on each node.
-        - name: install-cni
-          image: "{{ .Values.image }}"
+        - name: install-cni-v1_0
```

Comment:
Creation of this DaemonSet fails because the container name can't contain underscores.

Reply:
i should know better :/

Reply:
fixed